Sources/Controllers/AppleMusicController.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  AppleMusicController.swift
+//  ControlKit
+//
+
+import Control
+import SwiftUI
+
+/// Wrapper for ``Control/Playback/AppleMusic`` - use this to control playback from the Apple Music app and subscribe to its state.
+public final class AppleMusicController: ObservableObject, PlaybackController {
+    
+    @Published public var isPlaying: Bool = Control.Playback.AppleMusic.isPlaying
```

Comment:
Should it be modifiable from outside? 👀

```suggestion
    @Published public private(set) var isPlaying: Bool = Control.Playback.AppleMusic.isPlaying
```